Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -2461,8 +2461,8 @@ void WebProcessProxy::enableRemoteWorkers(RemoteWorkerType workerType, const Use
 
 void WebProcessProxy::markProcessAsRecentlyUsed()
 {
-    if (liveProcessesLRU().contains(this))
-        liveProcessesLRU().appendOrMoveToLast(this);
+    if (liveProcessesLRU().contains(*this))
+        liveProcessesLRU().appendOrMoveToLast(*this);
```

Comment:
Couldn’t we make a `moveToLast` function instead? It seems unfortunate to do two separate hash table lookups.

Reply:
We don't need the check for `contains`, I can drop.